pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -90,7 +92,9 @@ func (q *graceTerminateRSList) flushList(handler func(rsToDelete *listItem) (boo
 		}
 		if deleted {
 			klog.InfoS("Removed real server from graceful delete real server list", "realServer", name)
-			q.remove(rs)
+			if _, ok := q.list[rs.String()]; ok {
+				delete(q.list, rs.String())
+			}
```

Comment:
in golang deleting a non-existing key from a map is a nop, I think we can do 
```suggestion
		delete(q.list, rs.String())
```

Reply:
https://go.dev/doc/effective_go#maps

Reply:
Thanks, Done